packages/shared/src/mock/utils.ts

```diff
@@ -60,21 +56,3 @@ export function findMockEntity(path: string): ProvidedEntity | undefined {
   }
   return child;
 }
-
-/*
- * Get a mock dataset's value as an ndarray.
- * Allow type-casting via generic for convenience, as the type of the values is known.
- */
-// eslint-disable-next-line etc/no-misused-generics
-export function getMockDataArray<T = number>(path: string): NdArray<T[]> {
-  const entity = findMockEntity(path);
-  assertDefined(entity);
-  assertDataset(entity);
-  assertMockDataset(entity);
-
-  const { value } = entity;
-  assertArray(value);
-  assertArrayShape(entity);
-
-  return ndarray(value.flat(entity.shape.length - 1) as T[], entity.shape);
-}
```

Comment:
As explained in the PR description, since the factory functions in `mockValues` generate ndarrays, we no longer need this utility, which used to look for a dataset's dimensions in the mock metadata in order to be able to generate an ndarray with the right shape.